README.rst

```diff
@@ -76,19 +49,11 @@ and then running ``bin/buildout``
 Contribute
 ----------
 
-- Issue Tracker: https://github.com/collective/plone.initializer/issues
-- Source Code: https://github.com/collective/plone.initializer
-- Documentation: https://docs.plone.org/foo/bar
-
-
-Support
--------
-
-If you are having issues, please let us know.
-We have a mailing list located at: project@example.com
+- Issue Tracker: https://github.com/mekomsolutions/plone.initializer/issues
+- Source Code: https://github.com/mekomsolutions/plone.initializer
 
 
 License
 -------
 
-The project is licensed under the GPLv2.
+The project is licensed under the MIT licence.
```

Comment:
@enyachoke , is this possible to just change the licence of a forked project?

Reply:
The project was not forked. This was from the project generator so I think it is fine unless you think otherwise.

Reply:
This is fine with me. 👍🏻